pkg/common/utils.go

```
@@ -70,3 +71,65 @@ func SliceContainsString(origTargetString string, stringSlice []string, ignoreCa
 	}
 	return false, "", 0
 }
+
+// GoFakeIt Password generator does not guarantee inclusion of characters.
+// Using a custom random password gennerator with guaranteed inclusions (atleast) of lower, upper, numeric and special characters
+func GenerateRandomPassword(lower, upper, numeric, special bool, length int) string {
+	if length < 1 {
+		return ""
+	}
+
+	var password []rune
+	var required []rune
+	var allowed []rune
+
+	lowerChars := []rune("abcdefghijklmnopqrstuvwxyz")
+	upperChars := []rune("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	specialChars := []rune("!@#$%^&*()-_=+[]{}|;:',.<>?/")
+	numberChars := []rune("0123456789")
+
+	// Ensure inclusion from each requested category
+	if lower {
+		rand, _ := rand.Int(rand.Reader, big.NewInt(int64(len(lowerChars))))
```

I am not sure if we should ignore errors here.